configure.ac

```diff
@@ -569,6 +569,58 @@ if test x$enable_music_midi_timidity != xyes -a \
     AC_MSG_WARN([MIDI support disabled])
 fi
 
+if test x$enable_music_midi_timidity != xyes -a \
+        x$use_music_midi_native != xyes -a x$use_music_midi_native != xyes_cpp -a \
```

Comment:
The timidity lines somehow get duplicated: attached patch fixes that and tidies it up a bit more. [patch.txt](https://github.com/libsdl-org/SDL_mixer/files/8609463/patch.txt)